content/docs/apps/managing-teammates.md

```diff
@@ -11,13 +11,7 @@ If you're part of a team using cloud.gov, you can invite teammates to get cloud.
 
 First, send them an invite from the following link. (This may prompt you to log into your cloud.gov account first.)
```

Comment:
This is a minor comment, but now that there's only one link, we can rewrite this a bit to look more elegant.

![screen shot 2017-04-24 at 4 23 35 pm](https://cloud.githubusercontent.com/assets/391313/25362697/6206de4c-290a-11e7-9b6d-2aafb9b5dcaf.png)